cypress/e2e/landingSpec.cy.js

```diff
@@ -49,15 +49,38 @@ describe('Landing page after logging in spec', () => {
     cy.url('match', 'https://www.google.com/search?q=papers');
     cy.visit('http://localhost:3000/')
 
-    cy.get('[href="https://www.google.com/search?q=plus"]').click()
-    cy.url('match', 'https://www.google.com/search?q=plus');
-    cy.visit('http://localhost:3000/')
-
     cy.get('[href="https://www.google.com/search?q=account"]').click()
     cy.url('match', 'https://www.google.com/search?q=account');
     cy.visit('http://localhost:3000/')
 
   })
 
-
+  describe('DropDown Menu', () => {
+    it('has container for dropdown elements', () => {
+      cy.visit('http://localhost:3000/')
+      cy.get('button.m-auto > .m-auto').click()
+      cy.get('.dropDownPlus').should('exist');
+    })
+    it('should render dropdown menu with link to create new contact', () => {
+      cy.visit('http://localhost:3000/')
+      cy.get('button.m-auto > .m-auto').click()
+      cy.contains('Create New Contact').should('exist');
+      cy.get('.bg-cyan-600 > :nth-child(1) > a').click()
+      cy.url('match', 'http://localhost:3000/contacts/new');
+    })
+    it('should render dropdown menu with link to create new company', () => {
+      cy.visit('http://localhost:3000/')
+      cy.get('button.m-auto > .m-auto').click()
+      cy.contains('Create New Company').should('exist');
+      cy.get('.bg-cyan-600 > :nth-child(2) > a').click()
+      cy.url('match', 'http://localhost:3000/companies/new');
+    })
+    it('should render dropdown menu with link to create new contact', () => {
+      cy.visit('http://localhost:3000/')
+      cy.get('button.m-auto > .m-auto').click()
+      cy.contains('Create New Job Application').should('exist');
+      cy.get('.bg-cyan-600 > :nth-child(3) > a').click()
+      cy.url('match', 'http://localhost:3000/jobapplications/new');
+    })  
+  })
 })
```

Comment:
idk if i'm missing this in the tests but you are testing for the presence of dropdown elements, may be worth testing the isDropDownOpen state directly just to check it's being toggled as expected. I might of missed it in your code. Correct me if i'm wrong.